### PR TITLE
feat: Add an enterprise Helm chart

### PR DESCRIPTION
**Pull request by Gabriel-Ladzaretti**

### CODE REVIEW

File: helm-charts/mend-renovate-ee/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: mend-renovate-enterprise-edition
+version: 6.5.0
+appVersion: 6.5.0
+description: Mend Renovate Enterprise Edition
+home: https://github.com/mend/renovate-ce-ee
+sources:
+  - https://github.com/renovatebot/renovate
+icon: https://app.renovatebot.com/images/logo.png
```
Comment (PhilipAbed) [inline helm-charts/mend-renovate-ee/Chart.yaml:9]: @justo-mend do we want the renovate logo or the mend logo here?

File: helm-charts/mend-renovate-ee/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: mend-renovate-enterprise-edition
+version: 6.5.0
```
Comment (Gabriel-Ladzaretti) [inline helm-charts/mend-renovate-ee/Chart.yaml:3]:
```suggestion
version: 0.1.0
```

File: helm-charts/mend-renovate-ee/values.yaml [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+renovateServer:
+  image:
+    repository: ghcr.io/mend/renovate-ee-server
+    tag: 6.5.0
+    pullPolicy: IfNotPresent
+
+  # Additional worker env vars
+  extraEnvVars:
+    - name: MEND_RNV_MC_TOKEN
+      value: eyJhbGciOiJSUzUxMiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoianVzdG8tbWVuZC1sb25nIiwiaWF0IjoxNjk3OTU5ODQzLCJleHAiOjE3MDU3MzU4NDN9.CIau08QjSpvOL65gbsLMGAQBMGRO8MKPcH3zCZeuJUVlxSdof-ttSY6t51kTMkYfDrDSIMBA2jxWji94BJ2kLRj07LtrB6gRIJBrjJGDw4qVLmu8gTVBT9hcWoptwEltqtjMNE0kS2MhGPhWDiXqqnFBgVDTFsdWn7bDm895FP6_MGRdb-sdFa8HHsleN5IqKSJLWKMW2wGq59c7-6pwLUrNtaPP_nRTngukrkfxLZLwUxNjDbRxVC6bTTwRYnz_9fwfB98LenFVYLit5z8JtZCNOku8fppwCJUWbFm65bdzgremc1k01NMM2Ku4xmPBiv8gpACwC4GFEwP7Uo6DtM8NQOBpmH2t8rMRn-2hEJ6tHnDBFIQ0qA5N0LbhE46lKn-KFL1PXK1kbP9yLp5yCXL9fjFjd8lTUVqDojKc_CW-vVUlrdOqaw86xvsQiAn9KgCo7jqrIHssEcdF0K41keb3iOkUKx4n1TGQtigIqS7oUMZxlIAxG1-pcmcbD5o-b_85JOwCHF3-B19D5Z0HdfEyEZXIDNUZzO0qTvdnPMNQ7louRaUNExfMLg3w5EpIGHUijJLnmKaH2sjPdLkZyPruGRh5N1gknOuaZQo-cfJSt15taU1Hm-OukLjamLjbb9kPhUpxTLpNHWsCp2VpTUO4sisElrml_cMUnhJpiJw
```
Comment (PhilipAbed) [inline helm-charts/mend-renovate-ee/values.yaml:28]:
```suggestion
      value: eyJhbGciOiJSUzUxMiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoianVzdG8tbWVuZC1sb25nIiwiaWF0IjoxNjk3OTU5ODQzLCJleHAi...
```

File: helm-charts/mend-renovate-ee/values.yaml [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+  #   mendRnvLicenseKey:
+  existingSecret:
+
+renovateServer:
+  image:
+    repository: ghcr.io/mend/renovate-ee-server
+    tag: 6.5.0
+    pullPolicy: IfNotPresent
+
+  # Additional worker env vars
```
Comment (justo-mend) [inline helm-charts/mend-renovate-ee/values.yaml:25]:
```suggestion
  # Additional server env vars
```

**Review by justo-mend — CHANGES_REQUESTED** [review]

File: helm-charts/mend-renovate-ee/Chart.yaml (hunk 1 above)
Comment (justo-mend) [reply helm-charts/mend-renovate-ee/Chart.yaml:9]: Mend logo would be better.

Comment (justo-mend) [reply helm-charts/mend-renovate-ee/Chart.yaml:9]: This requires more investigation - to get the right perma-link for the right logo image. We will come back to this and update CE and EE charts together in a separate PR.

**Review by PhilipAbed — APPROVED** [review]

**Pull request merged**